docs/sphinx/examples/python/tutorials/vqe.ipynb

```diff
@@ -19,7 +19,63 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Similar to above, I think the outputs for this cell should be cleared. They add lots of text to the docs page without adding much value in my opinion.

And also similar to above, why are we changing this one under the "Adding AFQMC tutorial" PR?